long-ride-alerts/src/test/java/org/apache/flink/training/exercises/longrides/LongRidesUnitTest.java

```diff
@@ -104,6 +132,15 @@ public void shouldAlertOnWatermark() throws Exception {
         assertThat(harness.numEventTimeTimers()).isZero();
     }
 
+    private Long resultingRideId() {
+        ConcurrentLinkedQueue<Object> results = harness.getOutput();
+        assertThat(results.size())
+                .isEqualTo(1)
+                .withFailMessage("Expecting test to have exactly one result");
```

Comment:
From the Javadoc:
> You must set it before calling the assertion otherwise it is ignored as the failing assertion breaks the chained call by throwing an AssertionError.

```suggestion
        assertThat(results.size())
                .withFailMessage("Expecting test to have exactly one result")
                .isEqualTo(1);
```